collect/cache/cuckoo.go

```diff
@@ -50,13 +50,16 @@ var cuckooTraceCheckerMetrics = []metrics.Metadata{
 	{Name: CurrentLoadFactor, Type: metrics.Gauge, Unit: metrics.Percent, Description: "the fraction of slots occupied in the current cuckoo filter"},
 }
 
-func NewCuckooTraceChecker(capacity uint, m metrics.Metrics) *CuckooTraceChecker {
+func NewCuckooTraceChecker(capacity uint, addQueueDepth uint, m metrics.Metrics) *CuckooTraceChecker {
```

Comment:
`addQueueDepth` param isn't used, should it be used on line 62?

Reply:
Good catch 😮‍💨